samples/new-node_vagrant.yaml

```diff
@@ -0,0 +1,17 @@
+# Use this manifest to announce the new node to the apiserver.
+# You need to fill the needed SSH connection information.
+# CMD: sudo kubectl --kubeconfig /etc/kubernetes/admin.conf create -n "kube-system" -f new-node_vagrant.yaml
+apiVersion: v1
+kind: Node
+metadata:
+  name: node1
+  annotations:
+    metalk8s.scality.com/ssh-user: vagrant 
+    metalk8s.scality.com/ssh-port: 22
```

Comment:
Needs to be a string otherwise I get this:
```
Error from server (BadRequest): error when creating "new_node.yaml": Node in version "v1" cannot be handled as a Node: v1.Node.ObjectMeta: v1.ObjectMeta.Annotations: ReadString: expects " or n, but found 2, error found in #10 byte of ...|sh-port":22,"metalk8|..., bigger context ...|y_for_k8s_nodes","metalk8s.scality.com/ssh-port":22,"metalk8s.scality.com/ssh-sudo":"true","metalk8s|...
```